stacks/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: stacks
```

Comment:
If this is the same of the entry in the catalog, I'd directly call it `docker-compose`, since that's the value for the user.